### ISSUE TITLE
unable to find installable unit

### ISSUE THREAD

**Issue by s3dse** [diffplug/goomph#13]
Hi, 
I'm trying to use the asMaven plugin for several modeling tools (`org.eclipse.qvt`, `org.eclipse.ocl`, `org.eclipse.xtend`). When I run gradle I get the following error message: Unable to find: Installable Unit [ id=org.eclipse.ocl ]
I tried to verify the existence of the IUs via p2Director and each IU can be found.

`/Applications/Eclipse_4.6/Eclipse.app/Contents/MacOS/eclipse -application org.eclipse.equinox.p2.director    -repository http://download.eclipse.org/releases/neon -list 'Q:everything.select(x|x.id == "org.eclipse.ocl")'
org.eclipse.ocl=3.6.0.v20160523-1914`

When I try the dependencies in the examples (e.g. `org.eclipse.text`) it works. Am I missing something?

My `build.gradle` looks like this:

```
plugins {
    id 'com.diffplug.gradle.p2.asmaven' version '3.0.6'
}
apply plugin: 'java'

p2AsMaven {
    group 'eclipse-deps', {
        repoEclipse '4.6'
        iu          'org.eclipse.xtend'
    }
}

dependencies {
    compile 'eclipse-deps:org.eclipse.jdt.core:+'
}
```
Here is the stacktrace:
```
org.gradle.api.ProjectConfigurationException: A problem occurred configuring root project 'de.s3dse.tag.model'.
        at org.gradle.configuration.project.LifecycleProjectEvaluator.addConfigurationFailure(LifecycleProjectEvaluator.java:79)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:74)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:61)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:573)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:125)
        at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:42)
        at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:38)
        at org.gradle.initialization.DefaultGradleLauncher$2.run(DefaultGradleLauncher.java:151)
        at org.gradle.internal.Factories$1.create(Factories.java:22)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:53)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:148)
        at org.gradle.initialization.DefaultGradleLauncher.access$200(DefaultGradleLauncher.java:33)
        at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:112)
        at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:106)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:63)
        at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:106)
        at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:92)
        at org.gradle.launcher.exec.GradleBuildController.run(GradleBuildController.java:66)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:41)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:26)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:79)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:51)
        at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:59)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:47)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:26)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:34)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:74)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:72)
        at org.gradle.util.Swapper.swap(Swapper.java:38)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:72)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:60)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:72)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.HintGCAfterBuild.execute(HintGCAfterBuild.java:44)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:50)
        at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:293)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: com.diffplug.common.base.Errors$WrappedAsRuntimeException: java.lang.reflect.InvocationTargetException
        at com.diffplug.common.base.Errors.asRuntime(Errors.java:408)
        at com.diffplug.common.base.Errors.rethrowErrorAndWrapOthersAsRuntime(Errors.java:132)
        at com.diffplug.common.base.Errors$$Lambda$3/187465254.apply(Unknown Source)
        at com.diffplug.common.base.Errors$Rethrowing.lambda$wrap$11(Errors.java:344)
        at com.diffplug.common.base.Errors$Rethrowing$$Lambda$10/307661722.get(Unknown Source)
        at com.diffplug.common.base.Errors$Rethrowing.get(Errors.java:334)
        at com.diffplug.gradle.p2.AsMavenExtension.lambda$run$101(AsMavenExtension.java:58)
        at com.diffplug.gradle.p2.AsMavenExtension$$Lambda$7/1001593729.accept(Unknown Source)
        at com.diffplug.gradle.p2.AsMavenExtension.run(AsMavenExtension.java:53)
        at com.diffplug.gradle.p2.AsMavenPlugin.lambda$null$103(AsMavenPlugin.java:122)
        at com.diffplug.gradle.p2.AsMavenPlugin$$Lambda$5/1273010549.run(Unknown Source)
        at com.diffplug.common.base.Errors.lambda$wrap$5(Errors.java:218)
        at com.diffplug.common.base.Errors$$Lambda$6/1627485149.run(Unknown Source)
        at com.diffplug.common.base.Errors.run(Errors.java:210)
        at com.diffplug.gradle.p2.AsMavenPlugin.lambda$applyOnce$105(AsMavenPlugin.java:122)
        at com.diffplug.gradle.p2.AsMavenPlugin$$Lambda$1/330947098.execute(Unknown Source)
        at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:93)
        at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:82)
        at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:44)
        at org.gradle.internal.event.BroadcastDispatch.dispatch(BroadcastDispatch.java:79)
        at org.gradle.internal.event.BroadcastDispatch.dispatch(BroadcastDispatch.java:30)
        at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
        at com.sun.proxy.$Proxy15.afterEvaluate(Unknown Source)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:67)
        ... 52 more
Caused by: java.lang.reflect.InvocationTargetException
        at org.eclipse.ant.core.AntRunner.run(AntRunner.java:511)
        at org.eclipse.ant.core.AntRunner.start(AntRunner.java:608)
        at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
        at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
        at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
        at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher$Running.run(EquinoxLauncher.java:183)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher$Running.access$100(EquinoxLauncher.java:165)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher.run(EquinoxLauncher.java:156)
        at com.diffplug.gradle.eclipserunner.JarFolderRunner.run(JarFolderRunner.java:36)
        at com.diffplug.gradle.eclipserunner.JarFolderRunnerExternalJvm$RunOutside.run(JarFolderRunnerExternalJvm.java:98)
        at com.diffplug.gradle.JavaExecable.main(JavaExecable.java:134)
Caused by: /var/folders/k6/srk0k35j33d2sp_1kt8gryf40000gn/T/goomph-ant-build7461666184836541147.xml:2: Unable to find: Installable Unit [ id=org.eclipse.xtend ] 
        at org.eclipse.equinox.p2.internal.repository.tools.tasks.AbstractRepositoryTask.prepareIUs(AbstractRepositoryTask.java:187)
        at org.eclipse.equinox.p2.internal.repository.tools.tasks.MirrorTask.execute(MirrorTask.java:60)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:435)
        at org.apache.tools.ant.helper.ProjectHelper2.parse(ProjectHelper2.java:180)
        at org.eclipse.ant.internal.core.ant.InternalAntRunner.parseBuildFile(InternalAntRunner.java:387)
        at org.eclipse.ant.internal.core.ant.InternalAntRunner.run(InternalAntRunner.java:675)
        at org.eclipse.ant.internal.core.ant.InternalAntRunner.run(InternalAntRunner.java:568)
        ... 12 more
```

**Comment by nedtwigg** [comment]
Probably you need `feature 'org.eclipse.xtend'` instead of `iu`.  If that fixes it let me know, otherwise I'll dig in deeper.

**Comment by s3dse** [comment]
Thanks for your quick reply. Unfortunately it doesn't help. The feature cannot be found:
```
org.gradle.api.ProjectConfigurationException: A problem occurred configuring root project 'de.s3dse.tag.model'.
        at org.gradle.configuration.project.LifecycleProjectEvaluator.addConfigurationFailure(LifecycleProjectEvaluator.java:79)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:74)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.evaluate(LifecycleProjectEvaluator.java:61)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:573)
        at org.gradle.api.internal.project.DefaultProject.evaluate(DefaultProject.java:125)
        at org.gradle.execution.TaskPathProjectEvaluator.configureHierarchy(TaskPathProjectEvaluator.java:42)
        at org.gradle.configuration.DefaultBuildConfigurer.configure(DefaultBuildConfigurer.java:38)
        at org.gradle.initialization.DefaultGradleLauncher$2.run(DefaultGradleLauncher.java:151)
        at org.gradle.internal.Factories$1.create(Factories.java:22)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:53)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:148)
        at org.gradle.initialization.DefaultGradleLauncher.access$200(DefaultGradleLauncher.java:33)
        at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:112)
        at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:106)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:63)
        at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:106)
        at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:92)
        at org.gradle.launcher.exec.GradleBuildController.run(GradleBuildController.java:66)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:41)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:26)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:79)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:51)
        at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:59)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:47)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:26)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:34)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:74)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:72)
        at org.gradle.util.Swapper.swap(Swapper.java:38)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:72)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:60)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:72)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.HintGCAfterBuild.execute(HintGCAfterBuild.java:44)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:120)
        at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:50)
        at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:293)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: com.diffplug.common.base.Errors$WrappedAsRuntimeException: java.lang.reflect.InvocationTargetException
        at com.diffplug.common.base.Errors.asRuntime(Errors.java:408)
        at com.diffplug.common.base.Errors.rethrowErrorAndWrapOthersAsRuntime(Errors.java:132)
        at com.diffplug.common.base.Errors$$Lambda$3/187465254.apply(Unknown Source)
        at com.diffplug.common.base.Errors$Rethrowing.lambda$wrap$11(Errors.java:344)
        at com.diffplug.common.base.Errors$Rethrowing$$Lambda$10/307661722.get(Unknown Source)
        at com.diffplug.common.base.Errors$Rethrowing.get(Errors.java:334)
        at com.diffplug.gradle.p2.AsMavenExtension.lambda$run$101(AsMavenExtension.java:58)
        at com.diffplug.gradle.p2.AsMavenExtension$$Lambda$7/1001593729.accept(Unknown Source)
        at com.diffplug.gradle.p2.AsMavenExtension.run(AsMavenExtension.java:53)
        at com.diffplug.gradle.p2.AsMavenPlugin.lambda$null$103(AsMavenPlugin.java:122)
        at com.diffplug.gradle.p2.AsMavenPlugin$$Lambda$5/1273010549.run(Unknown Source)
        at com.diffplug.common.base.Errors.lambda$wrap$5(Errors.java:218)
        at com.diffplug.common.base.Errors$$Lambda$6/1627485149.run(Unknown Source)
        at com.diffplug.common.base.Errors.run(Errors.java:210)
        at com.diffplug.gradle.p2.AsMavenPlugin.lambda$applyOnce$105(AsMavenPlugin.java:122)
        at com.diffplug.gradle.p2.AsMavenPlugin$$Lambda$1/330947098.execute(Unknown Source)
        at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:93)
        at org.gradle.internal.event.BroadcastDispatch$ActionInvocationHandler.dispatch(BroadcastDispatch.java:82)
        at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:44)
        at org.gradle.internal.event.BroadcastDispatch.dispatch(BroadcastDispatch.java:79)
        at org.gradle.internal.event.BroadcastDispatch.dispatch(BroadcastDispatch.java:30)
        at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
        at com.sun.proxy.$Proxy15.afterEvaluate(Unknown Source)
        at org.gradle.configuration.project.LifecycleProjectEvaluator.notifyAfterEvaluate(LifecycleProjectEvaluator.java:67)
        ... 52 more
Caused by: java.lang.reflect.InvocationTargetException
        at org.eclipse.ant.core.AntRunner.run(AntRunner.java:511)
        at org.eclipse.ant.core.AntRunner.start(AntRunner.java:608)
        at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
        at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
        at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
        at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:380)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher$Running.run(EquinoxLauncher.java:183)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher$Running.access$100(EquinoxLauncher.java:165)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher.run(EquinoxLauncher.java:156)
        at com.diffplug.gradle.eclipserunner.JarFolderRunner.run(JarFolderRunner.java:36)
        at com.diffplug.gradle.eclipserunner.JarFolderRunnerExternalJvm$RunOutside.run(JarFolderRunnerExternalJvm.java:98)
        at com.diffplug.gradle.JavaExecable.main(JavaExecable.java:134)
Caused by: /var/folders/k6/srk0k35j33d2sp_1kt8gryf40000gn/T/goomph-ant-build3711590311527528373.xml:2: Unable to find: Installable Unit [ id=org.eclipse.xtend.feature.group ] 
        at org.eclipse.equinox.p2.internal.repository.tools.tasks.AbstractRepositoryTask.prepareIUs(AbstractRepositoryTask.java:187)
        at org.eclipse.equinox.p2.internal.repository.tools.tasks.MirrorTask.execute(MirrorTask.java:60)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:435)
        at org.apache.tools.ant.helper.ProjectHelper2.parse(ProjectHelper2.java:180)
        at org.eclipse.ant.internal.core.ant.InternalAntRunner.parseBuildFile(InternalAntRunner.java:387)
        at org.eclipse.ant.internal.core.ant.InternalAntRunner.run(InternalAntRunner.java:675)
        at org.eclipse.ant.internal.core.ant.InternalAntRunner.run(InternalAntRunner.java:568)
        ... 12 more
```

build.gradle:

```
plugins {
    id 'com.diffplug.gradle.p2.asmaven' version '3.0.6'
}
apply plugin: 'java'

p2AsMaven {
    group 'eclipse-deps', {
        repoEclipse '4.6'
        feature 'org.eclipse.xtend'
    }
}

dependencies {
    compile 'eclipse-deps:org.eclipse.jdt.core:+'
}
```

**Comment by nedtwigg** [comment]
This will fix it: replace `repoEclipse '4.6'` -> `repo 'http://download.eclipse.org/releases/neon'`

`repoEclipse '4.6'` resolves to `http://download.eclipse.org/eclipse/updates/4.6`, which is different (and much faster to resolve) than the full neon simultaneous release `http://download.eclipse.org/releases/neon`.  `repoEclipse` includes only the core eclipse platform and jdt.

If this is the only feature you need, I'd recommend finding it's own repository, rather than traversing through the maze of composite repositories in the simultaneous realease.  I helped somebody else traverse this maze [here on gitter](https://gitter.im/diffplug/goomph?at=57d34acf7b9f816711380e8e).

**Comment by s3dse** [comment]
Cool, it works. You're right: resolution takes a while. I'll look for the repositories of the features. Thank you!
